### PR TITLE
- Starting the structure.

### PR DESCRIPTION
**Pull request by etiennenoel**

### CODE REVIEW

File: docs/getting-started/01-overview/01.setup.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+Now, you probably have a very important question:
+
+> Ok, but how can I host it and server Http calls?
+
+You will learn this in the next section: [Hosting](02.hosting.md)
+
+
+### Real examples
+Here are some Open Source projects that are using Pristine:
+* [Pristine Business Intelligence Tool](https://github.com/magieno/pristine-business-intelligence-tool)
```
Comment (mathieugh) [inline docs/getting-started/01-overview/01.setup.md:131]: We could also link pristine-starter that I'll finish soon

File: docs/getting-started/01-overview/01.setup.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+const bootstrap = async () => {
+    const kernel = new Kernel();
+    await kernel.init(AppModule);
+}
+
+bootstrap();
+```
+
+#### Wrapping up
+That's it really, you now know how to initialize the Pristine Kernel. You could retrieve your services and have all the dependencies injected.
```
Comment (mathieugh) [inline docs/getting-started/01-overview/01.setup.md:120]: Maybe some explanation on dependency injection/ link to tsyringe ?

File: docs/getting-started/01-overview/02.hosting.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
+Hosting / Execution
+-------------------
+
+## How to host
+==============
```
Comment (mathieugh) [inline docs/getting-started/01-overview/02.hosting.md:5]: There's some weird formatting everywhere we have ==========

File: docs/getting-started/01-overview/02.hosting.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+In this section, we'll explore the different ways that Pristine can be hosted.
+
+You can host Pristine under different contexts:
+* Function as a Service (Serverless)
+* NodeJS Server (Container, Virtual machine or your local machine)
+* For scripting purposes by running it directly via the command line
+
+### Function as a Service (Serverless)
+======================================
+
```
Comment (mathieugh) [inline docs/getting-started/01-overview/02.hosting.md:15]: Maybe a comment saying we want to support Azure and GCP eventually so people don't get scared that we only work with AWS ?

File: docs/getting-started/01-overview/01.setup.md (hunk 1 above)
Comment (etiennenoel) [reply docs/getting-started/01-overview/01.setup.md:131]: Yes totally, I'll do it.

File: docs/getting-started/01-overview/01.setup.md (hunk 2 above)
Comment (mathieugh) [reply docs/getting-started/01-overview/01.setup.md:120]: I would still make a comment here saying : Yes we support dependency injection ! To learn more check out Dependency injection or tsyringe.

Comment (etiennenoel) [reply docs/getting-started/01-overview/01.setup.md:120]: yeah makes sense.

**Review by mathieugh — APPROVED** [review]

**Pull request merged**